### DISCUSSION TITLE
[20231105] Update not confirmed, endless database update

### DISCUSSION THREAD

**Discussion by kw-sudo** [rern/rAudio#1487] [Bug Reports]
Hello rern,

I just upgraded to version 20231105. After the process is complete, the button to update/install the addon is still "blue" (active) even if it is the current release version. At the same time, the database update runs in a continuous loop, even though I have deactivated the automatism in the settings. 

Oliver

RPi3B+

[edit]: downgraded to i20231022 - works

**Comment by Mightyflea64** [comment]
I have the same problem with this version.

RPi4b

**Reply by cYCL157** [reply]
Same

**Reply by ha11oga11o** [reply]
I have same/similar issue. I added NAS and trying to make database so i can browse files and actually create playlist; button is flashing last 2 days, nothing is happening. NAS icon when selected wants to recreate database gain. So basically since i installed rAudio to Rpi3 i play only couple songs from USB stick. NAS data is still creating database, but at /srv/http/data/mpd i see file "mpd.db" changed from few KB to 2.447KB and stays like that since 08.11.2023. I really dont know is it stuck, is it working, or its just failed and ignoring sanity. Database (bottom left bar) button is still flashing. 
At folder /mnt/MPD/NAS/NasData/Media/Music with Winscp i can see files, can browse, even copy them to my desktop. So it seems its mounted. 
Does anyone wave any idea whats happening and how can i check? I still cant find log location just to see live changes/errors. And i did also long hold update to do full upgrade of system.

Thanks in advance.

rAudio 20231022
6.1.58-2 aarch64
Raspberry Pi 3B+ Rev 1.3
BCM2837B0 • 906 MiB
4 x Cortex-A53 @ 1.40 GHz
0.09 • 0.04 • 0.01
39.7 °C
2023-11-10 • 21:54:56 UTC +00:00
2d 14h 9m since 2023-11-08 • 07:45

**Reply by kw-sudo** [reply]
Hello,

I also use a (Synology) NAS, but I'm not that technically proficient. For this reason, I chose the path of least resistance and simply reinstalled with version i20231022. 

I hope rern can do something with your findings. 

Oliver

**Reply by ha11oga11o** [reply]
Mine is fresh install. I did it let say 7 times, and have same problem.

**Comment by Raitsa** [comment]
5.11 updated to 11.11, now the update button is grey again but system infos still showing 5.11.

**Reply by EarsofTin** [reply]
For the last several updates I've had to reboot to have the correct version number display in the system page. No problems with the updates themselves.

**Comment by kw-sudo** [comment]
@rern

Thanks for fixing! For my purposes, with update 20231111 everything is working again.

Oliver
